org.opent2t.sample.hub.superpopular/com.wink.hub/js/manifest.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest>
+  <!-- associated voice handler -->
+  <voice id="org.opent2t.sample.lightcontrol.cortana.vcd.superpopular" />
+  <!-- associated schemas -->
+  <schemas>
+      <schema id="org.opent2t.sample.thermostat.superpopular" main="true" type="OCF" />
+      <schema id="oic.r.temperature" type="OCF" />
```

Comment:
This shouldn't be here, since the hub schema should not reference temperature (only core and baseResource should be sufficient)